sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -33,6 +33,10 @@
 #include "modules/computer_vision/viewvideo.h"
 #include "modules/computer_vision/cv.h"
 
+#include "modules/computer_vision/video_thread.h"
+
+#include "boards/bebop.h"
```

Comment:
Mmm, you are right that this is a problem. Looks like the cameras of the ARDrone are described in board.c... and the bebop cameras in video.c.... Is there a way to include a specific file depending on the platform you are compiling for?